content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx

```diff
@@ -14,7 +14,37 @@ Prisma 4 introduces a number of **breaking changes** when you upgrade from an ea
 
 ## Breaking changes
 
-This section gives an overview of breaking changes in Prisma 4, grouped under [general changes](#general-changes) that affect both the Prisma Schema and the Prisma Client, [Client changes](#client-changes) and [Schema changes](#schema-changes).
+This section gives an overview of breaking changes in Prisma 4, grouped under [general changes](#general-changes) that affect both the Prisma Schema and the Prisma Client, [Schema changes](#schema-changes) and [Client changes](#client-changes).
+
+We recommend to first address any Prisma schema validation errors, then pull your database to reflect new Prisma schema capabilities and finally fix any type errors in client and validate by running your test suite. 
+
+### Upgrade your Prisma Schema
+
+1. carefully skim the list of changes and check if you are impacted by a breaking change
```

Comment:
Nitpick: should this all be capitalized? Do we put full stops at the end of list items in the docs?

Reply:
done